docker-compose.metrics.yml

```diff
@@ -0,0 +1,26 @@
+version: '2.1'
+services:
+  graphite:
+    image: sitespeedio/graphite
+    ports:
+        - "8181:80"
+
+  statsite:
+    image: quay.io/azavea/statsite:0.7.1
```

Comment:
I'd like to try and swap this out for the `azavea/statsd` image so that we can more easily support Librato w/ the Tags API. OK if I make those changes here?